crds/exoscale.appcat.vshn.io_exoscalekafkas.yaml

```diff
@@ -87,10 +87,11 @@ spec:
                           type: object
                           x-kubernetes-preserve-unknown-fields: true
                         version:
-                          default: "3.2"
-                          description: Version contains the minor version for Kafka. Currently only "3.2" is supported. Leave it empty to always get the latest supported version.
+                          default: "3.4"
```

Comment:
out of curiosity - wouldn't that affect running instances - if any?

Reply:
Yes it  would, but currently there are no Kafkas on APPUiO Exoscale and at this point users can't actually provision one as 3.2 is not available anymore...